tox.ini

```diff
@@ -66,3 +66,9 @@ commands =
 [pycodestyle]
 ignore = E203,E731,W503,W504,C0330,
 max-line-length = 100
+
+[gh-actions]
+python =
+  3.8: py38, lint
```

Comment:
we already have this in the gh actions

Reply:
where is it? I usualy put it here, but it does not work indeed, I wanted to add the docs test in the ci

Reply:
found it!